# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: 62f8ce89dce8
# Full Hash: 62f8ce89dce8f1162ff2f57debdd9a756a11a2f9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-03 01:02:14
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r bcbee7191130 -r 62f8ce89dce8 gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Thu Apr 02 01:38:21 2020 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Thu Apr 02 01:38:18 2020 +0000
@@ -770,7 +770,8 @@
   LangPack = 2,  // From an optional OS component such as language support
   User = 3,      // User-installed font (or installed by another app, etc)
   Hidden = 4,    // Internal system font, should never exposed to users
-  Count = 5,     // Count of values, for IPC serialization
+  Webfont = 5,   // Webfont defined by @font-face
+  Count = 6,     // Count of values, for IPC serialization
 };
 
 namespace IPC {
@@ -944,6 +945,9 @@
 
   FontVisibility Visibility() const { return mVisibility; }
   bool IsHidden() const { return Visibility() == FontVisibility::Hidden; }
+  bool IsWebFontFamily() const {
+    return Visibility() == FontVisibility::Webfont;
+  }
 
  protected:
   // Protected destructor, to discourage deletion outside of Release():